server.js

```diff
@@ -1,27 +1,117 @@
 import express from "express";
 import cors from "cors";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
+
+import netflixData from "./data/netflix-titles.json";
 // import topMusicData from "./data/top-music.json";
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
+
+
 const port = process.env.PORT || 8080;
 const app = express();
-
+const listEndpoints = require('express-list-endpoints');
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json(listEndpoints(app));
+});
+app.get('/titles', (req, res) => {
+  res.json(netflixData)
+  } )
+
+
+app.get('/title/:id', (req, res) => {
+  const { id } = req.params;
+  const showId = netflixData.filter((title) => {
+    return title.show_id === Number(id);
+  });
+  if (showId.length) {
+    res.status(200).json({
+      success: true,
+      message: "OK",
+      body: {
+        title: showId
+      }
+    });
+  } else {
+    res.status(404).json({
+      success: false,
+      message: "title not found",
+      body: {}
+    });
+  }
+});
```

Comment:
This works as it should! If you add to your condition to check if showID.length is 8 characters long, then you can send different error messages for wrongful id (eg. too many/few numbers) code 400, and an error for correct id-input but no matching id (404). Just a small addition, not necessarily needed.

`if (id.length !== 8) {
    res.status(400).json({
      success: false,
      message: "Invalid ID length. The ID should be 8 characters long.",
      body: {}
    });`